internal/pkg/template/templates/workloads/services/lb-web/manifest.yml

```diff
@@ -16,6 +16,18 @@ http:
   path: '{{.HTTPOrBool.Main.Path}}'
   # You can specify a custom health check path. The default is "/".
   # healthcheck: '{{.HTTPOrBool.Main.HealthCheck.Basic}}'
+  {{- if gt (len .HTTPOrBool.AdditionalRoutingRules) 0}}
+  # You can specify additional rules to route customer traffic to different container ports.
+  additional_rules:
+    {{- range $i, $rule := .HTTPOrBool.AdditionalRoutingRules }}
+    - path: '{{$rule.Path}}'
+      target_port: {{$rule.TargetPort}}
+      {{- if $rule.ProtocolVersion }}
+      version: {{$rule.ProtocolVersion}}
+      {{- end }}
+      # healthcheck: '{{$rule.HealthCheck.Basic}}'
```

Comment:
I think it's there to make healthcheck functionality visible since it's not super intuitive that you could turn it on. But we don't want to turn it on by default.

Reply:
I think since we're not adding multiple ports and rendering this by default i'm going to remove it.